src/usng.ts

```diff
@@ -1339,12 +1339,12 @@ extend(Converter.prototype, {
       return 0;
     }
 
-    strregexp = new RegExp("^[0-9]{2}[CDEFGHJKLMNPQRSTUVWX]$");
+    strregexp = new RegExp("^([1-9]|[1-5][0-9]|60)[CDEFGHJKLMNPQRSTUVWX]$");
     if (usngStr.match(strregexp)) {
       return 0;
     }
 
-    strregexp = new RegExp("^[0-9]{2}[CDEFGHJKLMNPQRSTUVWX][ABCDEFGHJKLMNPQRSTUVWXYZ][ABCDEFGHJKLMNPQRSTUV]([0-9][0-9]){0,5}$");
+    strregexp = new RegExp("^([1-9]|[1-5][0-9]|60)[CDEFGHJKLMNPQRSTUVWX][ABCDEFGHJKLMNPQRSTUVWXYZ][ABCDEFGHJKLMNPQRSTUV]([0-9][0-9]){0,5}$");
```

Comment:
It looks like usng4j has ```\s[0-9]{0,5}\s[0-9]{0,5}``` for the coordinate part of the regex instead. Which implementation do we want? https://github.com/codice/usng4j/pull/11/files#diff-a692ac7744aec78853858fe4f9ad7592R48

Reply:
This is a fair point, but out of scope for this pull request.